src/syscall.c

```diff
@@ -18,21 +18,23 @@
  * system call: name, number
  */
 /* clang-format off */
-#define SUPPORTED_SYSCALLS       \
-    _(close,            57)      \
-    _(lseek,            62)      \
-    _(read,             63)      \
-    _(write,            64)      \
-    _(fstat,            80)      \
-    _(exit,             93)      \
-    _(gettimeofday,     169)     \
-    _(brk,              214)     \
-    _(clock_gettime,    403)     \
-    _(open,             1024)    \
-    IIF(RV32_HAS(SDL))(          \
-        _(draw_frame,   0xBEEF)  \
-        _(setup_queue,  0xC0DE)  \
-        _(submit_queue, 0xFEED), \
+#define SUPPORTED_SYSCALLS           \
+    _(close,                57)      \
+    _(lseek,                62)      \
+    _(read,                 63)      \
+    _(write,                64)      \
+    _(fstat,                80)      \
+    _(exit,                 93)      \
+    _(gettimeofday,         169)     \
+    _(brk,                  214)     \
+    _(clock_gettime,        403)     \
+    _(open,                 1024)    \
+    IIF(RV32_HAS(SDL))(              \
+        _(setup_audio,      0xD00D)  \
```

Comment:
Again, don't mix audio and sound in mnemonic identifiers. `setup_audio` and `control_audio` should appear at the end of syscall listing.